README.md

```diff
@@ -1,5 +1,18 @@
 # Wortels
-This is the design repo for wortels, a specification for some random project.
+
+This is the design repo for Wortels - a novel design for a consensus computer.
+
+It preliminary takes learnings from Ethereum.
+
+The goals are:
+- can operate as a pegged chain or sidechain
+- stateless chain
+- no native token (aka "account abstraction")
+- WebAssembly as an execution engine (this should focus in linking code)
+- no compatibility with EVM is strived for by the design
```

Comment:
Maybe this should not be a design goal. It instead could be noted as
"Compatibility with Ethereum's EVM is not a primary goal of the project."